frontend/src/components/ViewActivities/activity.js

```diff
@@ -1,25 +1,98 @@
 import React from 'react';
-import Card from 'react-bootstrap/Card';
 import * as time from '../Utils/time.js';
 import * as DB from '../../constants/database.js'
 import '../../styles/activities.css';
-import Accordion from 'react-bootstrap/Accordion';
+import * as autils from './activityfns.js';
```

Comment:
Maybe call them activityUtils? I got confused initially because my mind immediately went to "authUtils" lol.